Apps/W1/HybridGP/app/src/tables/GPMigrationLog.Table.al

```diff
@@ -0,0 +1,49 @@
+table 41006 "GP Migration Log"
```

Comment:
is this table cleared before each migration? if not, should there be date\time and\or some other way to determine what migration run each record is for?

Reply:
It's cleared for the company when the company is deleted (to do a migration again). What is seen in the log would be for the most recent run for the company.